.github/workflows/3x.yml

```diff
@@ -134,7 +134,7 @@ jobs:
     if: ${{ startsWith(github.ref, 'refs/tags/')  && github.repository == 'pantheon-systems/terminus' }}
     steps:
       - name: Download terminus.phar as artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
This already failed. That was the first error. Actually I think to get the release out we should downgrade back to v3 everything and then update later

Reply:
Agreed.